lib/xml2js.js

```diff
@@ -313,9 +328,13 @@ function onError(error) {
   error.note = error; //console.error(error);
 }
 
-module.exports = function (xml, userOptions) {
+var parser;
 
-  var parser = pureJsParser ? sax.parser(true, {}) : parser = new expat.Parser('UTF-8');
+module.exports = function (xml, userOptions) {
+  var saxOptions = userOptions && userOptions.position ? {
+    position: userOptions.position
+  } : {};
+  parser = pureJsParser ? sax.parser(true, saxOptions) : parser = new expat.Parser('UTF-8');
```

Comment:
I guess it should work regardless, but is the extra assignment in the else part purposeful? 
I'd think it should be
```
parser = pureJsParser ? sax.parser(true, saxOptions) : new expat.Parser('UTF-8');
```